### DISCUSSION TITLE
[Feature] Support sqlite

### DISCUSSION THREAD

**Discussion by webysther** [immich-app/immich#1701] [Feature Request]
### The feature

For small deployments will be great support slite as storage.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by zackpollard** [comment]
We currently have no intention of support SQLite as it isn't suited to the kind of operations we need to use a database for.

**Comment by bo0tzz** [comment]
Sqlite has been discussed previously and support is currently not planned. See https://github.com/immich-app/immich/issues/562
